cli_reference/openshift_developer_cli/installing-odo.adoc

```diff
@@ -7,12 +7,13 @@ toc::[]
 
 The following section describes how to install `{odo-title}` on different platforms.
 
+Currently, `{odo-title}` does not support installation in a restricted network environment.
+
 [NOTE]
 ====
-Currently, `{odo-title}` does not support installation in a restricted network environment.
+You can always find the URL to the latest binaries in the web console of your cluster. To access them using the web console; click the *?* icon in the upper-right corner of the window, and select *Command Line Tools* in the drop-down menu.
```

Comment:
s/always/also

Reply:
Suggest same change in the Helm module too.

Reply:
I think the placement is good. The other option is to add it to the end of the section. But I think the current option tells users that this section has CLI instructions, but if he wants to use the console he can access it before reading all of the CLI instructions.